styles/scss/_font.scss

```diff
@@ -3,4 +3,45 @@
 body {
     font-family: "Source Sans Pro", sans-serif;
     font-size: 16px;
+    font-weight: 300;
+}
+
+.reset-font {
+    font-size: 1rem;
+    font-weight: 300;
+}
+
+.text-bold {
```

Comment:
rename text-xxx classes to font-xxx or rename the file to text.scss but then reset-font should be renamed to text-reset.